src/url-handling.js

```diff
@@ -215,6 +222,23 @@ export function stateFromLocation(location: Location): UrlState {
   };
 }
 
+function toValidTabSlug(slug: ?string): TabSlug {
+  switch (slug) {
+    case 'calltree':
+    case 'stack-chart':
+    case 'marker-chart':
+    case 'marker-table':
+    case 'summary':
+      return slug;
+    default:
+      console.error(
+        'Unknown tab found, maybe a URL upgrader needs to be written.',
+        slug
+      );
```

Comment:
This goes here when we access the root, eg http://localhost:4242/

So maybe you should have a  `case undefined` returning `calltree` which is a valid case, so that an error is not displayed in the console.